src/app/tracing/io/data-importer/data-importer-v0.ts

```diff
@@ -181,14 +179,14 @@ export class DataImporterV0 implements IDataImporter {
                     contains: e.contains
                 });
             }
-            const weight = e.weight != null ? e.weight : (e.outbreak ? 1 : 0);
+            const weight = e.weight ?? e.outbreak ? 1 : 0;
```

Comment:
This replacement might yield a different value.
Please change to `const weight = e.weight ?? (e.outbreak ? 1 : 0);`

Reason:
The nullish coalescing operator has a higher precedence than the ternary operator.
So a `e.weight` of `0.1` would originally result in `weight = 0.1`.
Your replacement would lead to `weight = 1`.

Remark:
That's why I usually keep the brackets. Because I always forget about the operator precedences.

Reply:
Interesting, I didn't consider that. Thanks.

I've tested this and as you've said:
- 0.1 ?? true ? 1 : 0;
=> 1
-  0.1 ?? (true ? 1 : 0);
=>  0.1